app/test_main.py

```diff
@@ -1,3 +1,37 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "value1,value2,result",
+    [
+        (-3, -3, [0, 0]),
+        (14, 14, [0, 0]),
+        (16, 16, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17]),
+        (32, 94, [4, 16])
+    ]
```

Comment:
Add test case for 23 cat/dog age
```suggestion
@pytest.mark.parametrize(
    "value1,value2,result",
    [
        (-3, -3, [0, 0]),
        (14, 14, [0, 0]),
        (16, 16, [1, 1]),
        (24, 24, [2, 2]),
        (23, 23, [1, 1]),
        (100, 100, [21, 17]),
        (32, 94, [4, 16])
    ]
```